requirements.txt

```diff
@@ -1 +1,2 @@
 .[all]
+git+git://github.com/moloney/dcmstack@master
```

Comment:
```suggestion
git+git://github.com/moloney/dcmstack@fa24702#egg-dcmstack
```

Reply:
You may also have to install this first - looks like its getting overwritten on install
https://travis-ci.org/nipy/heudiconv/jobs/652087701#L718